client/starwhale/core/dataset/tabular.py

```diff
@@ -98,30 +98,18 @@ def save_to_datastore(self, ds_wrapper: DatastoreWrapperDataset) -> None:
 
 class TabularDatasetRow(ASDictMixin):
 
-    ANNOTATION_PREFIX = "annotation/"
+    DATA_PREFIX = "data/"
```

Comment:
this line will make existed dataset fail to load.

Reply:
This pr is breaking change anyway, existing old version dataset is expected fail to be loaded.